apps/meteor/client/views/room/contextualBar/Apps/AppsWithData.tsx

```diff
@@ -46,7 +47,7 @@ const useValues = (view: IUIKitSurface): [any, Dispatch<any>] => {
 	}));
 
 	const initializer = useMutableCallback(() => {
-		const filterInputFields = (block: IBlock): boolean => {
+		const filterInputFields = (block: IBlock | Block): boolean => {
```

Comment:
sorry to ask but why this change is important?

Reply:
The latest `alpha` of Apps-Engine includes a change where we started accepting `Block`s directly from `@rocket.chat/ui-kit` as a peer dependency. However, this caused the types in these callbacks to be invalid and breaking the CI.

It seems this "new" dependency was added to the Apps-Engine before but it had not been updated in Rocket.Chat yet. Then, when we updated the Apps-Engine to support this regression, the dependency changes came along.